data/blog/2024-year-in-review.mdx

```diff
@@ -0,0 +1,545 @@
+share our year in review. We want to look back at the announcements we made,
+share an update regarding the donations we received, and give an overview of
+how we allocated our funds. After that we will share updates to our grantee
+statistics and provide an outlook of what the next year might bring.
+
+- [Announcements](#announcements)
+- [Donations](#donations)
+- [Allocation](#allocation)
+- [Grantees](#grantees)
+- [Outlook](#outlook)
+
+But before we dive into the various graphs and numbers, let us spend a minute to
+remind everyone—and most importantly ourselves—_why_ we are doing what we are
+doing.  _"I'm convinced that to give away a dollar effectively is harder than to
+make a dollar,"_ Steve Jobs once mentioned in an interview. And indeed, making
+sure that grants are given to the right projects at the right time—in other
+words: making sure that [grant selection](/selection) is done fairly and
+properly—is an incredibly hard problem.
+
+Why set out to solve this problem in the first place? Why go through the trouble
+of creating a non-profit entity? Why spend countless hours reviewing grant
+applications, commits, reports, pull requests, and other software artifacts? Why
+bother with open-source grants at all? Shouldn't this problem be solved by
+companies, or direct crowd funding, or the state?
+
+From the beginning, the vision of OpenSats was to create a sustainable
+foundation that provides [consistent funding](/about) for free and open-source
+projects, especially those who make bitcoin flourish. We don't want to be
+dependent on quarterly profits nor on funding from the state, which is why we
+chose a public charity as a vehicle.
+
+As mentioned in our [mission statement](/mission), we believe that we can help
+make bitcoin more accessible, robust, secure, and decentralized. Further, we
+believe that the projects we support have the potential to revolutionize the way
+we think about individual sovereignty and liberty—values that we hold in high
+regard. We want to operate with as little overhead as possible, which is why we
+don't take a cut from donations. And we don't want any one person
+or small group to have control over funding or organizational decisions, which
+is why we have a 9-person board that is the final arbiter of all decisions.
+
+We don't claim to have all the answers, and we don't claim that what we are
+doing is frictionless or perfect. We are bound to make mistakes, and we will
+always be prone to false positives (funding projects that aren't as fruitful as
+we thought they would be) as well as false negatives (not funding projects that
+should receive funding). We strive to gradually and continually improve our
+processes, and we hope to get better at what we are doing with every passing
+year. That said, we are incredibly pleased to see that our structure and
+processes are working, and we believe this year in review is a testament to
+that.
+
+---
+
+## Announcements
```

Comment:
To avoid situations that announce announcements, I recommend renaming this section something like "Scope of our Grants" or "Scope and Focus of Our 2024 Grants".

Reply:
If you want to stick to one word titles, consider calling this section "Grantees" and renaming the other section with the same name as "Growth".